classes/class-loader.php

```diff
@@ -336,12 +336,15 @@ public function enqueue_editor_scripts( $hook ) {
 		$en_active = ! MigrationLog::from_wp_options()->already_ran( M001EnableEnFormFeature::get_id() )
 					|| Features::is_active( Features::ENGAGING_NETWORKS );
 
+		$uses_cdn = is_plugin_active( 'wp-stateless/wp-stateless-media.php' ) && get_option( 'sm_custom_domain' );
```

Comment:
I had to change the check to this because we use a constant to set the mode, and [it's always `stateless](https://github.com/search?q=org%3Agreenpeace+%22WP_STATELESS_MEDIA_MODE%22&type=code)`.